code/datums/components/sign_language.dm

```diff
@@ -53,6 +51,8 @@
 	// Sign language is toggled on/off via adding/removing TRAIT_SIGN_LANG.
 	RegisterSignal(parent, SIGNAL_ADDTRAIT(TRAIT_SIGN_LANG), PROC_REF(enable_sign_language))
 	RegisterSignal(parent, SIGNAL_REMOVETRAIT(TRAIT_SIGN_LANG), PROC_REF(disable_sign_language))
+	linked_action.Grant(parent)
```

Comment:
was this a merge skew moment? else, SR edit comments are needed

Reply:
This is a change from upstream, which is a hot-fix. The TG maintainers introduced this bug into the code last-minute and then merged it after I did some minimal testing. The same thing will likely show up in a mirror PR